Gems/AudioSystem/Code/Source/Engine/ATLComponents.h

```diff
@@ -161,6 +144,23 @@ namespace Audio
         float m_fTimeSinceLastVelocityUpdateMS;
 
         AudioRaycastManager m_raycastManager;
+
+#if !defined(AUDIO_RELEASE)
```

Comment:
What defines AUDIO_RELEASE and why not just check _RELEASE?

Reply:
We can defer changing this for a later time.  It evolved from a definition that essentially meant "NOT_RELEASE", flipped it to this version that simply means "RELEASE".  It could be cleaned up at some point.